src/controllers/users.controller.js

```diff
@@ -0,0 +1,60 @@
+const usersServices = require('../services/users.services');
+const { asyncHandler } = require('../utils/asyncHandler');
+const { getErrorWithStatus } = require('../utils/getError');
+const { getValidId, getValidString } = require('../utils/validation');
+
+const getUsers = async (_, res) => {
+  const users = await usersServices.getUsers();
+
+  res.status(200).json(users);
+};
+
+const getUserById = async (req, res) => {
+  const userId = getValidId(req.params.id, 'User ID is invalid');
+
+  const user = await usersServices.getUserById(userId);
+
+  if (!user) {
+    throw getErrorWithStatus(404, 'User not found');
+  }
+
+  res.status(200).json(user);
+};
+
+const createUser = async (req, res) => {
+  const name = getValidString(req.body.name, 'name');
+
+  const newUser = await usersServices.createUser(name);
+
+  return res.status(201).json(newUser);
+};
+
+const removeUser = async (req, res) => {
+  const userId = getValidId(req.params.id, 'User ID is invalid');
+
+  const user = await usersServices.getUserById(userId);
+
+  if (!user) {
+    throw getErrorWithStatus(404, 'User not found');
+  }
+  await usersServices.removeUser(+userId);
```

Comment:
Ensure that the `userId` is consistently used as a string or number throughout the code to avoid potential type-related issues. Consider removing the `+` operator if `userId` should remain a string.